cmd/lncli/commands.go

```diff
@@ -2845,11 +2845,10 @@ var listInvoicesCommand = cli.Command{
 			Name:  "max_invoices",
 			Usage: "the max number of invoices to return",
 		},
-		cli.BoolTFlag{
-			Name: "reversed",
-			Usage: "if set, the invoices returned precede the " +
-				"given index_offset, allowing backwards " +
-				"pagination",
+		cli.BoolFlag{
+			Name: "paginate-forwards",
+			Usage: "if set, invoices succeeding the " +
+				"index_offset will be displayed",
```

Comment:
nit: s/be displayed/be returned/ to stay consistent with the wording in the other descriptions.

Reply:
Agreed, have changed it :+1: